src/wp-admin/menu.php

```diff
@@ -210,7 +210,7 @@
 	$submenu['themes.php'][6] = array(
 		__( 'Template Parts' ),
 		'edit_theme_options',
-		'site-editor.php?postType=wp_template_part',
+		'site-editor.php?postType=wp_template_part&path=/wp_template_part/all',
```

Comment:
```suggestion
		'site-editor.php?postType=wp_template_part&amp;path=/wp_template_part/all',
```

Should be `&amp;` instead of `&` WDYT @Mamaduka?

Reply:
Good point. Updated.